allensdk/test/brain_observatory/behavior/test_stimulus_processing.py

```diff
@@ -1,52 +1,141 @@
-
+import pandas as pd
 import numpy as np
 import pytest
 
-
 from allensdk.brain_observatory.behavior.stimulus_processing import (
-    get_stimulus_presentations, _get_stimulus_epoch, _get_draw_epochs)
-
-
-data = {
-    "items": {
-        "behavior": {
-            "stimuli": {
-                "images": {
-                    "set_log": [
-                        ('Image', 'im065', 5.809955710916157, 0),
-                        ('Image', 'im061', 314.06612555068784, 6),
-                        ('Image', 'im062', 348.5941232265203, 12),
-                    ],
-                    "draw_log": ([0]+[1]*3 + [0]*3)*3 + [0]
-                }
-            },
-            # "intervalsms": np.array([16.0]*10)
+    get_stimulus_presentations, _get_stimulus_epoch, _get_draw_epochs,
+    get_visual_stimuli_df, get_stimulus_metadata)
+
+
+@pytest.fixture()
+def behavior_stimuli_time_fixture(request):
+    """
+    Fixture that allows for parameterization of behavior_stimuli stimuli
+    time data.
+    """
+    timestamp_count = request.param.get("timestamp_count", 22)
+    time_step = request.param.get("time_step", 0.016)
+
+    fixture_params = {
```

Comment:
why is this being constructed and returned? It isn't used anywhere.

Reply:
Removed